scripts/Source/sslSystemConfig.psc

```diff
@@ -760,7 +760,7 @@ endFunction
 bool function CheckSystem()
 	; Check Skyrim Version
 	if !CheckSystemPart("Skyrim")
-		CheckSkyrim.Show()
+		CheckSkyrim.Show(1.5)
```

Comment:
To fix the issue with the Skyrim version message together with the modification to show the value on message text 0x70C3A in the SexLab.esm.